lib/xcake/command/init.rb

```diff
@@ -0,0 +1,18 @@
+module Xcake
+  class Command
+    class Init < Command
+
+      self.summary = "Initilises an example Cakefile for creating a project"
+      self.description = "Initilises an example Cakefile for creating a project"
+
+      def run
+
+        cakefile_path = "#{File.dirname(__FILE__)}/../resources/Cakefile"
+        cakefile_contents = File.read(cakefile_path)
+        File.write("Cakefile", cakefile_contents)
+
+        puts "Open Cakefile to edit and run xcake bake to get your xcode project"
```

Comment:
Line is too long. [81/80]<br>Do not write to stdout. Use Rails' logger if you want to log.